add 2024 changes (#20)
ubcuas/ubcuas.com: content/english/pages/recruitment.md

{{< /columns >}}

{{% markdown class="text-center pt-4" %}}
## Fall Recruitment Timeline
{{% /markdown %}}

{{% markdown class="text-center pt-4" %}}
### Upcoming Events

Application Window: August 25th - September 15th

Come visit us at one of the upcoming events to ask questions and learn more about UBC Uncrewed Aircraft Systems!
{{% /markdown %}}

{{< columns containerClass="pt-2 pb-6 flex flex-row flex-wrap justify-center gap-y-4 items-start" itemClass="flex-initial basis-64">}}

{{% markdown %}}
#### Imagine Day

- **Date:** September 3rd
- **Time:**  1:00 - 4:30 PM
- **Location:** Main Mall
  - TBD
    {{% /markdown %}}

<--->

{{% markdown %}}
#### Info Session 1

- **Date:** September 4th
- **Time:** 6:00 - 8:00 PM
- **Info:** Presentation and Q&A
- **Location:** TBD
    {{% /markdown %}}

<--->

{{% markdown %}}
#### Info Session 2

- **Date:** September 10th
- **Time:** 6:00 - 8:00 PM
- **Info:** Presentation and Q&A
- **Location:** TBD
    {{% /markdown %}}
    {{< /columns >}}

{{% markdown class="text-center pt-4" %}}
## Position Openings
{{% /markdown %}}

{{< toc >}}

{{% markdown class="text-center pt-4 pb-3" %}}
## General Member Roles
{{% /markdown %}}

> UBC UAS has three subteams for general members: **Aircraft**, **Payload**, and **Software**. Each subteam has divisions that focus on specific skillsets within the subteam.

Reviewer: ```suggestion
UBC UAS has three sub-teams for general members: **Aircraft**, **Payload**, and **Software**. Each sub-team has divisions that focus on specific skillsets within the sub-team.
```

{{% markdown class="text-center pt-4 pb-3" %}}
### Aircraft Roles

> The aircraft subteam builds innovative uncrewed aerial vehicles for our competitions. We work closely with our software and payload subteams to ensure proper integration between our systems. We are also responsible for the upkeep of active aircraft and ensuring flight readiness for test flights and competition.

Reviewer: ```suggestion
The aircraft sub-team builds innovative uncrewed aerial vehicles for our competitions. It works closely with our software and payload sub-teams to ensure proper integration between our systems. The aircraft sub-team is also responsible for the upkeep of active aircraft and ensuring flight readiness for test flights and competition.
```

{{% /markdown %}}

#### MECH Aircraft

Design and manufacture aerodynamic and lightweight airframes as well as control surfaces and landing gear. Members will learn and apply skills in mechanical design, aerodynamics, manufacturing, control surface selection, airfoil analysis and simulation tools applicable across industries.

#### ELEC Aircraft

Design of electrical systems within our aircraft, working closely with software and payload to ensure proper integration between systems. Members will learn and apply skills that are essential across industries including communication systems management, power management, signal processing, soldering, PCB design, and PID tuning and sensor calibration.

{{% markdown class="text-center pt-4 pb-3" %}}
### Payload Roles

> The payload subteam creates mission-specific payload systems for competition. The subteam works on projects including autonomous delivery rovers, winch mechanisms, and anything capable of interfacing with the aircraft. These projects involve the integration of multiple engineering disciplines.

Reviewer: ```suggestion
The payload sub-team creates mission-specific payload systems for competition. The sub-team works on projects including autonomous delivery rovers, winch mechanisms, and anything capable of interfacing with the aircraft. These projects involve the integration of multiple engineering disciplines.
```

{{% /markdown %}}

#### MECH Payload

Work alongside electrical designers to develop enclosures and structures controlled by electrical components. You will make use of CAD, FEA software, and various manufacturing techniques including 3D printing and laser cutting. Knowledge of statics, materials, and mechanical design principles is an asset.

#### ELEC Payload

Design segments of payload projects with electrical aspects. You will learn to use microcontrollers to control motors and circuits, using EDA, RC, power distribution, etc. to ensure proper control and movements of the payload components.

{{% markdown class="text-center pt-4 pb-3" %}}
### Software Roles

> The software team assigns roles based on preference and need, where members works on many different projects/programs and uses a variety of tools and languages. We encourage members to work on projects that they are interested in and apply their experience while learning new tools.

Reviewer: ```suggestion
The software team assigns roles based on preference and need, where members works on many different projects/programs and uses a variety of tools and languages. Members are encouraged to work on projects that they are interested in and apply their experience while learning new tools.
```

{{% /markdown %}}

#### Software Member

> All projects are up for grabs (everyone gets to choose at the start of the year what they get to work on) and we plan to work with a sprint based system. Some languages and tools we use: JavaScript, Go, C++, Git, Angular, Python, React, Django, Docker, Linux CLI. The Some developers may work on Payload subteam projects, which requires embedded development skills, including the use of microcontrollers, Git, control systems, telemetry, and more. Knowledge of motor control, control systems, and microcontrollers is an asset. Any knowledge with these tools and languages are a plus. Prior professional software development experience is not required but recommended. Open source contribution is also a plus.

Reviewer: ```suggestion
All projects are up for grabs (everyone gets to choose at the start of the year what they get to work on) and the team works on a sprint-based system. Some languages and tools the team uses: JavaScript, Go, C++, Git, Angular, Python, React, Django, Docker, and Linux. Some developers may work on Payload sub-team projects, which require embedded development skills, including the use of microcontrollers, Git, control systems, telemetry, and more. Knowledge of motor control, control systems, and microcontrollers is an asset. Any knowledge with these tools and languages is a plus. Prior professional software development experience is not required but strongly recommended. Open-source contributions are also a plus.
```

{{% markdown class="text-center pt-4 pb-3" %}}
### Admin Team Roles
{{% /markdown %}}

#### Events Coordinator

The events coordinator is responsible for organizing and planning team events. This includes team socials, workshops, and trips. The events coordinator will also be responsible for organizing and planning public events such as recruitment events. This role is a great opportunity to develop your leadership, communication, and organizational skills.

#### Finance Manager

> The finance manager is responsible for managing the team’s finances. This includes managing the team’s budget, applying for grants, managing sponsorships, and delegating finance work to executive members. This role is a great opportunity to develop your financial knowledge, leadership, and organizational skills.

Reviewer: ```suggestion
The finance manager is responsible for managing the team’s finances. This includes managing the team’s budget, applying for grants, processing purchases, and delegating finance work to executive members. This role is a great opportunity to develop your financial knowledge, leadership, and organizational skills.
```

The content editor is responsible for writing and proofreading all written content produced by the team is clear, engaging, and free of errors. This includes editing social media updates any other written communications. The role involves close collaboration with the social media manager, website designer, and other team members. It’s an excellent opportunity to hone your writing, editing, and communication skills.

{{% markdown class="text-center pt-4 pb-3" %}}
## FAQ
{{% /markdown %}}

{{< accordion title="When do applications open?" >}}
You will be able to access the application through our website starting August 25th at midnight (PST).
{{< /accordion >}}

{{< accordion title="Where do I apply?" >}}
This webpage (ubcuas.com/recruitment)! All the application details and links will be updated on August 25th.
{{< /accordion >}}

{{< accordion title="What do I need to apply?" >}}
The application process includes a general submission (with your resume, desired position, as well as some other information that will be released on August 25th). Be ready to show your eagerness and passion!
{{< /accordion >}}

{{< accordion title="What are the different roles?" >}}

> There are openings for our 3 subteams: Aircraft, Payload, and Software. Within the Aircraft and Payload teams, we have divisions for ELEC and MECH, with tasks within those divisions being related to their discipline:

Reviewer: ```suggestion
There are openings for our 3 sub-teams: Aircraft, Payload, and Software. Within the Aircraft and Payload teams, we have divisions for ELEC and MECH, with tasks within those divisions being related to their discipline:
```
